### ISSUE TITLE
convergence retraction nystagmus 

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7134]
**Preferred term label:**
convergence retraction nystagmus 

**Synonyms**

**Definition (free text, please give PubMed ID)**
-- does this require a new term?

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://pubmed.ncbi.nlm.nih.gov/23836506/  1/4
OMIM:617008 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added term
